src/views/Play/Play.tsx

```diff
@@ -73,6 +73,9 @@ export class Play extends React.Component<PlayProperties, any> {
         if (w !== this.seekgraph.width || h !== this.seekgraph.height) {
             this.seekgraph.resize(w, h);
         }
+        if (w === 0 || h === 0) { // Wait for positive size
```

Comment:
Honestly I don't *know* for sure if there's ever been a case where I saw `< 0`, but I've been bitten enough times over the years by doing things like checking for zero when what I really want is to ensure the value is positive that I just check for `<= 0` or `> 0`  by default now.